MAUI/LinearGauge/mirror-linear-gauge.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: Mirror in .NET MAUI Linear Gauge control | Syncfusion
+description: Learn here about mirroring the Syncfusion .NET MAUI Linear Gauge (SfLinearGauge) control with IsMirrored property
+platform: maui
+control: SfLinearGauge
+documentation: ug
+---
+
+# Mirrored in .NET MAUI Linear Gauge (SfLinearGauge)
+
+The [`IsMirrored`]() property in [`SfLinearGauge`]() mirrors all the Gauge elements in the `SfLinearGauge`. The following code sample demonstrates how to setting the [`IsMirrored`]() property.
```

Comment:
The [`IsMirrored`]() property in [`SfLinearGauge`]() mirrors all Gauge elements in the `SfLinearGauge`. The following code sample demonstrates how to set the [`IsMirrored`]() property.